.github/workflows/integration-tests.yml

```diff
@@ -369,6 +369,24 @@ jobs:
             nodes: 1
             os: ubuntu-latest
             pyroscope_env: ci-smoke-forwarder-ocr-evm-simulated
+          - name: log-poller-finality-tag
```

Comment:
Two questions:
1. Is it possible to run these tests in parallel?
2. If the answer to the above is no then do we think we will need to add more log poller tests? We have a setup using json files to handle test files with multiple docker based tests that helps us notice when a new test is added but not added to CI with that. With only 3 tests it might be overkill to move to that but if we are going to be adding any more we might want to go that route.

Reply:
@tateexon correct me if I'm wrong, but they do run in parallel already, no? Each single test in a dedicated runner. We could add 3 more tests to that matrix to always run both fixed and finality tag-using tests, but idk if that's required and how much value would that bring. Wdyt @reductionista?

Reply:
1. in parallel as in a separate job that runs parallel to this one? I think it should be, but unless we decide we want to run all 6 smoke tests not just 3 of them, then imho it's not needed.
2. @reductionista do you think it makes sense to run 3 tests we have (regular one, replay, chaos) for both fixed and finality tag versions or it's enough if we run any of these 3 tests with fixed depth and rest with finality tag?

Reply:
I don't see much point in running any of these tests in CI yet until we add a way to disable backup logpoller in config.  I guess we can run them, but we may get a lot of false positives.

Reply:
I think once we are ready to run all of them, we should probably run all 6... because there could be different bugs with fixed depth and finality tag for any of those situations

Reply:
In the end I did what you suggested, @tateexon 
![image](https://github.com/smartcontractkit/chainlink/assets/20401585/f6342c9e-074d-4974-9a30-87a31776a188)

@reductionista I think it's better to run them even if they are not testing LP in isolation, because backup poller is covering whatever issues we might have, we might still catch something.